src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -71,15 +98,51 @@ public void close() throws Exception {
     // Mostly for JUnit tests
     motor.close();
     motor2.close();
-    
   }
 
-  @Override
-  public void periodic() {
-    intakeMode.setString((isConeMode) ? "Cone Mode" : "Cube Mode");
-    // This method will be called once per scheduler run
+  public double getConeDist() {
+    return timeOfFlightSensor.getRawDistance0().classDistance;
   }
 
+  public double getCubeDist() {
+    return timeOfFlightSensor.getRawDistance1().classDistance;
+  }
+
+  public void changeFlightState() {
```

Comment:
This seems like it should be private, or just combined into the getFligthState() method

Reply:
Combined into getFlightState